index.html

```diff
@@ -216,6 +216,59 @@ <h1>Skills</h1>
                 </div>
             </div>
         </section>
+        <section class="contact-me">
+            <img id="contact-background" src="./images/mobile_geometry_1.svg" alt="background-box">
+            <div class="contact-me-form">
```

Comment:
Wonderful work implementing most of the requirements. Your project is great, but it can be better.

- [ ] Kindly change this to a `background-image`. If an image is a design, it is best to use it as a `background-image` instead of an actual image. It is a rule of thumb to use an actual image `(img)` if it has relevant data connected to the project that you are working on. 👍